### ISSUE TITLE
No test cases for the project

### ISSUE THREAD

**Issue by mr-nano** [lostfields/vscode-nodejs-repl#18]
I forked the repo and I realized that there are not test cases for the plugin. If one is serious for the plugin development, we should start adding the test cases.

**Comment by lostfields** [comment]
Yeah, I know, I will do it but the project started as proof of concept and I'm not entirely happy with the code base. Some refactoring is needed.

**Comment by mr-nano** [comment]
Yes. It might be a good idea to start writing tests, in that case we can be sure refactoring doesn't break stuff?

**Comment by lostfields** [comment]
Sure, but i'll want to implement a better base class for the REPL to make it easier to implement other languages such as Typescript

Pointless to write a test for the structure now, but I will do it soon
